<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>451</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabGeneral">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Template</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbTemplate">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>211</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leURL">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leServer">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Url</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Trainer Server</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSaveGeneral">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>180</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbMessage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbSIPTransport">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Force TCP transport for SIP</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabSip">
    <attribute name="title">
     <string>Audio</string>
    </attribute>
    <widget class="QComboBox" name="cbInput">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbOutput">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbRing">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>271</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Input Device</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Output Device</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ring Device</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbRingTest">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>80</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leRingFile">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>361</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="tone_stream://%(2000,4000,440.0,480.0);loops=20"/>
     </property>
     <property name="text">
      <string>tone_stream://%(2000,4000,440.0,480.0);loops=20</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>122</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ring File</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tbSelectFile">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>120</y>
       <width>25</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Choose a custom .wav file, we don't support mp3 right now.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;For default ring, set to: tone_stream://%(2000,4000,440.0,480.0);loops=20&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>263</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="2">
       <widget class="QPushButton" name="pbSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pbLoopTest">
        <property name="text">
         <string>Loop Test</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pbReset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Diagnostic</string>
    </attribute>
    <widget class="QCheckBox" name="cbDebug">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable Debug</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
